### ISSUE TITLE
<Base> cannot be used optionally for global styles when `disableAggregateTranslations` is true

### ISSUE THREAD

**Issue by bansalrachita** [cerner/terra-core#3447]
# Bug Report

## Description
When trying to import just the `<Base>` component in the app, and webpack config is `disableAggregateTranslations` set to true, the <Base> component seems to always log the error with `console.error` preventing the `children` to render. 

From the documentation it seems like the `<Base>` component is used to help manage translations and setting minimal global base styles. Optionally the internationalization feature can be turned off using `disableAggregateTranslations` flag in the env. Should the `<Base>` component then be used only for applying top level stylings?

## Steps to Reproduce
1. Set `disableAggregateTranslations` flag using the env in webpack config.
2. install the `terra-base` and import `<Base>` at the top level of the app.
```
<Base locale={locale} throwOnI18nLoadError={false}>
        {children}
 </Base>
```
3. Returns a `console.error` and prevent `children` to render.

## Additional Context / Screenshots
![image](https://user-images.githubusercontent.com/10090750/119632001-4643fd80-bdc5-11eb-826d-3faa4c7e4cf5.png)

## Expected Behavior
Should not log the error when `throwOnI18nLoadError` is set to false (by default) and allow the children to render.

## Environment
* Component Name and Version: Base, version: 5.46.0
* Browser Name and Version: Chrome, version: 90.0.4430.212
* Node Version: node 16
* Webpack Version: Webpack 5
* Operating System and version (desktop or mobile): MacOS

**Comment by mjhenkes** [comment]
Hi @bansalrachita, The aggregate translations function must be run prior to or during webpack for your application. The aggregate translation function will search through node modules and pull together the translations used by your app. terra-base and any other terra component with translations expects this step.

By default our webpack config aggregates translations when run, we have the disableAggregateTranslations flag for instances where consumers want to run aggregate translations themselves before packing the application.

Aggregate translations isn't my favorite pattern, but it is what we have until we can design something better.

You should also consider upgrading to use [terra-application-base](https://engineering.cerner.com/terra-ui/application/terra-application/components/application-base) the same translation rules apply but we are moving away from recommending consumers directly use terra-base and instead use terra-application-base.
